scripts/core/menu/views/about.html

```diff
@@ -11,11 +11,15 @@ <h3 class="modal__header-sd-logo">Superdesk</h3>
       <span class="sf-logo"></span>
       <h4>Superdesk {{ :: version }}</span></h4>
       <h5 ng-if="build_rev && build_rev !== ''">Superdesk rev. <a href="https://github.com/superdesk/superdesk/commit/{{ :: build_rev }}" rel="noopener noreferrer">{{ :: build_rev }}</a></h5>
-      <h5 ng-if="build_rev_core && build_rev_core !== ''">Superdesk core rev. <a href="https://github.com/superdesk/superdesk-core/commit/{{ :: build_rev_core }}" rel="noopener noreferrer">{{ :: build_rev_core }}</a></h5>
-      <h5 ng-if="build_rev_client && build_rev_client !== ''">Superdesk client rev. <a href="https://github.com/superdesk/superdesk-client-core/commit/{{ :: build_rev_client }}" rel="noopener noreferrer">{{ :: build_rev_client }}</a></h5>
+      <h5 ng-if="build_rev_core.href">Superdesk Core rev. <a ng-href="{{ :: build_rev_core.href }}" rel="noopener noreferrer">{{ :: build_rev_core.name }}</a></h5>
+      <h5 ng-if="build_rev_client.href">Superdesk Client rev. <a href="{{ :: build_rev_client.href }}" rel="noopener noreferrer">{{ :: build_rev_client.name }}</a></h5>
+      <h5 ng-if="build_rev_planning.href">Superdesk Planning rev. <a href="{{ :: build_rev_planning.href }}" rel="noopener noreferrer">{{ :: build_rev_planning.name }}</a></h5>
```

Comment:
Don't hardcode the names of plugins. The core should be agnostic to what plugins you are going to use. Make plugins provide their versions on registration and list them here dynamically.